Sources/Build/llbuild.swift

```diff
@@ -13,6 +13,21 @@ import Utility
 import PackageModel
 import PackageGraph
 
+/// An enum representing the different build options.
+public enum BuildOption {
+    /// Represents the option to build all targets and products (excluding test targets).
+    case allExcludingTests
+
+    /// Represents the option to build all targets and products (including test targets).
+    case allIncludingTests
+
+    /// Represents the option to build a specific target.
+    case target(ResolvedTarget)
+
+    /// Represents the option to build a specific product.
+    case product(ResolvedProduct)
```

Comment:
Tool doesn't need to resolve it, it'll pass the strings as-is. This will also allow other libSwiftPM clients to work with the API in an easy way.